cadc-storage-permissions/src/main/java/org/opencadc/inventory/permissions/PermissionsClient.java

```diff
@@ -89,17 +88,33 @@ public PermissionsClient() {
     }
     
     /**
-     * Get the permissions information about the file identified by fileURI.
+     * Get the read permissions information about the file identified by fileURI.
      * 
      * @param artifactURI Identifies the artifact for which to retrieve grant information.
-     * @return The grant information.
+     * @return The read grant information.
      * 
      * @throws ResourceNotFoundException If the file could not be found.
      * @throws TransientException If an unexpected, temporary exception occurred. 
      */
-    public Grant get(URI artifactURI) throws ResourceNotFoundException, TransientException {
-        // TODO
-        return new Grant(artifactURI);
+    public ReadGrant getReadGrant(URI artifactURI)
+        throws ResourceNotFoundException, TransientException {
+
+        throw new UnsupportedOperationException("Not implemented");
     }
-    
+
+    /**
+     * Get the read-write permissions information about the file identified by fileURI.
+     *
```

Comment:
change comment here to say 'write', not 'read-write'

Reply:
Also, you still need to provide a way to set annonymousAccess.  If it's mandatory it should be a param in the constructor...

Reply:
Changed and setAnonymousAccess method added.